source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -4,7 +4,7 @@ Use the mixer tool
 ##############
```

Comment:
Since your PR follows those before on same issue, please add `#` to include full length here.